assets/companies/components/EditCompanyDetails.jsx

```diff
@@ -81,6 +81,14 @@ export function EditCompanyDetails({company, companyTypes, users, errors, onChan
                     error={errors ? errors.contact_email : null}
                 />
 
+                <TextInput
```

Comment:
Should we hide this input if SSO is not enabled, as there would be no need to show an input that has no affect.